docs/dev-manuals/architecture/experiment-configuration.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 3
+---
+
+# Experiment configuration
+
+MisArch is intended as a microservice reference architecture.
+As such, it is possible to deterministically require certain conditions, i.e. a specific memory usage per pod.
+The problem is here that every service can be deployed \<x\> times simultaenously in the form of replicas.
+As such, the architecture to achieve this cannot be as simple as offering an endpoint to configure these limits on each service.
```

Comment:
Is that because a service does not know about its existing replicas and therefore could only inform its "own" instances / pods? If I got this right, I think it would make sense to add an explaining sentence like that in order to make it easier to grasp.